src/Models/Domain.php

```diff
@@ -30,6 +30,8 @@ public function __construct(
         public ?bool $dnsSec = null,
         public ?string $premiumPrice = null,
         public ?DateTimeInterface $executionAt = null,
+        public ?int $transferCode = null,
```

Comment:
I believe the transfer code should be a string. For most TLDs, it does not only contain digits.